package.json

```diff
@@ -16,9 +17,11 @@
   "license": "ISC",
   "dependencies": {
     "@types/node": "^14.14.20",
+    "cypress": "^6.5.0",
     "lezer-python": "^0.13.1",
     "lezer-tree": "^0.13.0",
     "mochawesome": "^6.2.1",
+    "python-shell": "^2.0.3",
```

Comment:
Why do you need python-shell?

Reply:
Going to use python-shell in typescript to evaluate generated python programs! After generating a python program, we would evaluate the correctness of the program by comparing the output of the compiled python-to-wasm code (by running the compiled wasm code) and the generated python code (using python-shell).

Example use case of python-shell:
`
import {PythonShell} from 'python-shell';

PythonShell.runString('Insert Python Program As String Here', null, function (err) {
  if (err) throw err;
  console.log('finished');
});
`